doc/containers.rst

```diff
@@ -0,0 +1,46 @@
+.. _containers:
+
+Containers
+==========
+
+We provide a pyPESTO container image in the OCI format through `docker.io/stephanmg/pypesto` (https://hub.docker.com/r/stephanmg/pypesto).
+
+Develop versions are pushed as tag ``latest``.
+
+The image can also be built using the provided `Dockerfile <https://github.com/ICB-DCM/pyPESTO/blob/main/docker/Dockerfile>`_.
+
+If you manually build your container and need to convert to Apptainer, first export your docker image to `.tar` by:
+`docker save -o pypesto.tar pypesto:latest` and pull the image before from docker.io by `docker pull docker.io/stephanmg/pypesto:latest`.
+
+Then execute for generating a `.sif` file:
+``apptainer build pypesto.sif docker-archive://pypesto.tar``
+
+Likewise for singularity:
+``singularity build pypesto.sif docker-archive://pypesto.tar``
+
+
+Docker
+------
+
+To download / pull image: `docker pull docker.io/stephanmg/pypesto`.
+
+To run: `docker run --rm pypesto python3 -c 'import pypesto; print(f"PyPESTO version: {pypesto.__version__}")'`.
+
+
+Apptainer
+---------
+
+Same principle as for Docker, but replace with `apptainer` commands:
```

Comment:
Would remove that, if we still show the commands used in apptainer this is if in the best case an interesting but not useful information in a "how to" imo

Reply:
Okay.